ivy/functional/frontends/torch/tensor.py

```diff
@@ -1313,6 +1313,11 @@ def square(self):
     def log10(self):
         return torch_frontend.log10(self._ivy_array)
 
+    @with_unsupported_dtypes({"2.0.1 and below": ("float16",)}, "torch")
```

Comment:
Correct me if I'm wrong but if `log10` and `log10_` are essentially the same function could you just not make one an alias of the other?

Reply:
Thank you for the suggestion! However, "log10" and "log10_" have different behaviors - one is in-place, and the other creates a new tensor. Making them aliases might not be suitable in this case.

Reply:
> Thank you for the suggestion! However, "log10" and "log10_" have different behaviors - one is in-place, and the other creates a new tensor. Making them aliases might not be suitable in this case.

Oh my bad! I must've missed that in the docs.